src/app/components/dropdown/index.html

```diff
@@ -11,61 +11,78 @@
   </sky-demo-page-summary>
 
   <sky-demo-page-properties
-    sectionHeading="Dropdown button properties">
+    sectionHeading="Dropdown properties">
     <sky-demo-page-property
-      propertyName="buttonType"
+      propertyName="alignment"
       isOptional="true"
-      defaultValue="select">
-      Specifies the type of button to render as the dropdown's trigger element. To display a button with text and a caret, specify <stache-code>select</stache-code> and then enter the button text in a <stache-code>sky-dropdown-button</stache-code> element. To display a round button with an ellipsis, specify <stache-code>context-menu</stache-code>. And to display a button with <a href="http://fontawesome.io/icons/">a Font Awesome icon</a>, specify the icon's class name. For example, to display the <stache-code>fa-filter</stache-code> icon, specify <stache-code>filter</stache-code>.
+      defaultValue="center">
+      Specifies the horizontal alignment of the dropdown menu in relation to the dropdown button. Available values are <stache-code>left</stache-code>, <stache-code>right</stache-code>, and <stache-code>center</stache-code>.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="buttonStyle"
       isOptional="true"
       defaultValue="default">
-      Specifies a background color for the dropdown button. If unspecified, the <stache-code>default</stache-code> value sets the background color to <stache-code>sky-color-white</stache-code>. To change the background color to <stache-code>sky-color-blue</stache-code>, specify <stache-code>primary</stache-code>.
+      Specifies a background color for the dropdown button. Available values are <stache-code>default</stache-code> and <stache-code>primary</stache-code>. These values set the background color from the <a routerLink="../button">secondary and primary button classes</a> respectively.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="buttonType"
+      isOptional="true"
+      defaultValue="select">
+      Specifies the type of button to render as the dropdown's trigger element. To display a button with text and a caret, specify <stache-code>select</stache-code> and then enter the button text in a <stache-code>sky-dropdown-button</stache-code> element. To display a round button with an ellipsis, specify <stache-code>context-menu</stache-code>. And to display a button with <a href="http://fontawesome.io/icons/">a Font Awesome icon</a>, specify the icon's class name. For example, to display the <stache-code>fa-filter</stache-code> icon, specify <stache-code>filter</stache-code>. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="disabled"
+      isOptional="true"
+      defaultValue="false">
+      Indicates whether to disable the dropdown button. This property accepts <stache-code>boolean</stache-code> values.
```

Comment:
You removed the "button" on the section heading. Is it still appropriate here?

Reply:
I think so. I removed "button" from the header because these properties are in SkyDropdownComponent at https://github.com/blackbaud/skyux-popovers/blob/master/src/app/public/modules/dropdown/dropdown.component.ts. The component name doesn't include "button," so I don't think it should be in the header. However, the properties do affect the dropdown button that the component creates, so I think it's fine to refer to the "dropdown button" to distinguish the button from the menu. (Note, I kept "menu" in the next header because it is part of the SkyDropdownMenuComponent name at https://github.com/blackbaud/skyux-popovers/blob/master/src/app/public/modules/dropdown/dropdown-menu.component.ts.)

Reply:
Sounds good. Just wanted to check!